docs/docs/user/getting-started/quick-start.md

```diff
@@ -0,0 +1,169 @@
+# Quick-start
+
+## Word-count
+
+Word-count is a demo pipeline which consists of producing words to Kafka, counting the number of times each word occurs and consuming them into a database.
```

Comment:
the first part of the sentence is missing a word. consists of ... (a producer?) producing words to Kafka

Reply:
"consuming" doesn't seem right in this context.
perhaps
... and export the results into a Redis database.

Reply:
Agreed, lmk what you think of the sentence now

Reply:
it reads very well now!